sdk/python/kfp/dsl/types/type_annotations.py

```diff
@@ -135,15 +135,15 @@ def is_Input_Output_artifact_annotation(typ) -> bool:
     return True
 
 
-def is_input_artifact(typ) -> bool:
+def is_artifact_wrapped_in_Input(typ: Any) -> bool:
```

Comment:
Note for reviewer: These renamings reduce ambiguity about the behavior of these functions in context of the current changes. Should `is_input_artifact` return `True` for the annotation `Artifact` in the component definition `def comp(artifact: Artifact)`? With the new name, I think it is clear that the answer is no.